release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -247,6 +248,8 @@ private void onTextChange(string newText)
 		this.objUnityInput.text = newText;
 		if (this.objUnityInput.onValueChanged != null)
 			this.objUnityInput.onValueChanged.Invoke(newText);
```

Comment:
Why not just remove these two lines, making the original event fire once instead of twice?

Reply:
I didn't really want to change the existing behaviour to not break anything depending on that. I'm not sure if this is a case that's happening only on some specific devices or someone else depends on this event even though it's triggered twice.

Reply:
Looking at the source code of `InputField`, it's clear that `onValueChanged` will be fired when the text is changed:
https://bitbucket.org/Unity-Technologies/ui/src/932f8d245259170afbc37f3cb0abf38b84a54546/UnityEngine.UI/UI/Core/InputField.cs?at=2018.2.18f1&fileviewer=file-view-default#lines-347

I think it's better to remove this duplication than to work around it. If we have code that depends on the buggy behaviour, that should be fixed.

Reply:
Please also remove it from here:
https://github.com/YousicianGit/UnityNativeEdit/blob/d5203e87c6d2faffcdb01f29ab134a446ea47e70/release/NativeEditPlugin/scripts/NativeEditBox.cs#L258-L259